cfg/tf2ware_ultimate/minigames.cfg

```diff
@@ -46,6 +46,7 @@ most_bombs
 move
 parachute
 eat_plate
+pick_up_that_can
```

Comment:
Change the name to "pickup_can" to match the two-word file name convention of every minigame